gmso/formats/lammpsdata.py

```diff
@@ -168,16 +170,18 @@ def write_lammpsdata(topology, filename, atom_style='full'):
 
             # TODO: Write out multiple dihedral styles
             if topology.dihedrals:
-                data.write('\nDihedral Coeffs\n')
+                data.write('\nDihedral Coeffs\n\n')
                 for idx, dihedral_type in enumerate(topology.dihedral_types):
-                    if dihedral_type.name == 'RyckaertBellemansTorsionPotential':
+                    rbtorsion = PotentialTemplateLibrary()['RyckaertBellemansTorsionPotential']
+                    if (dihedral_type.expression == sympify(rbtorsion.expression) or
+                        dihedral_type.name == rbtorsion.name):
                         dihedral_type = convert_ryckaert_to_opls(dihedral_type)
                     data.write('{}\t{:.5f}\t{:5f}\t{:5f}\t{:.5f}\n'.format(
                         idx+1,
-                        dihedral_type.parameters['k1']/2,
-                        dihedral_type.parameters['k2']/2,
-                        dihedral_type.parameters['k3']/2,
-                        dihedral_type.parameters['k4']/2
+                        dihedral_type.parameters['k1'].in_units(u.Unit('kcal/mol')).value,
```

Comment:
The docstring says it supports different `atom_style`s other than `full`. Is that true in this case, since we are converting to `kcal/mol`?

Reply:
@rmatsum836

Reply:
Yes, the `atom_style` defines the information that gets written out to the data file: https://lammps.sandia.gov/doc/atom_style.html.  Currently we only support the `real` style of units in lammps which uses energy units of `kcal`.